llvm-general/src/LLVM/General/Internal/OrcJIT.hs

```diff
@@ -172,3 +172,23 @@ withObjectLinkingLayer f =
     FFI.createObjectLinkingLayer
     FFI.disposeObjectLinkingLayer $ \objectLayer ->
       f (ObjectLinkingLayer objectLayer)
+
+createObjectLinkingLayer :: IO ObjectLinkingLayer
```

Comment:
Document all these functions with links to the underlying C++ (or language ref, as appropriate), as with all other exposed functions.